resources/repos.json

```diff
@@ -1104,5 +1104,23 @@
                 "1": "<a class=\"link_info\" href=\"/builds/TheBusyBiscuit/Slimefun4/master/#73\">dev #73</a>"
             }
         }
+    },
+    "kii-chan-reloaded/GeneticChickengineering:master": {
+        "sonar": {
+            "enabled": false
+        },
+        "options": {
+            "prefix": "DEV",
+            "abandoned": false,
+            "createJar": true
+        },
+        "dependencies": {
+            "Minecraft Version(s)": {
+                "1": "1.16.x"
+            },
+            "Slimefun Version": {
+                "1": "<a class=\"link_info\" href=\"/builds/TheBusyBiscuit/Slimefun4/stable/#20\">RC-20</a>"
+            }
```

Comment:
Pretty sure your plugin requires Paper to be used, right?
If so, then you should add this as a dependency here.